### ISSUE TITLE
Issue about training Kinetics400

### ISSUE THREAD

**Issue by fuyukaiKUN** [yjxiong/tsn-pytorch#87]
Hi, do anyone try training TSN on Kinetics400? 

I train the TSN using default hyperparameter setting given in the paper (i.e., lr=0.001, lr_steps=[30,60], epochs=80, batch_size=128) but only get a validation accuracy 64.88% on split1, which is much lower than the accuracy reported on "http://yjxiong.me/others/kinetics_action/". 

Why? Did  I use the wrong hyperparameter setting? How to get such high validation accuracy?

**Comment by LiuChaoXD** [comment]
I am training tsn-pytorch on kinetics400 data set.
However, I get the accuracy 33% for train set while accuracy 5% for validate set. Did you complete the experiment for  kinetics400?

**Comment by fuyukaiKUN** [comment]
Not yet

**Comment by yojayc** [comment]
My testing result on kinetics is 

Testing Results: Prec@1 44.483 Prec@5 70.912 Loss 2.53040

It is also quite low accuracy compare to the reported results

**Comment by cy-kui** [comment]
hello, i need the kinetics dataset, can you send me that? thank you!  2115996745@qq.com

**Comment by fuyukaiKUN** [comment]
> hello, i need the kinetics dataset, can you send me that? thank you! [2115996745@qq.com](mailto:2115996745@qq.com)
The original dataset is so large, maybe you can try this "https://pytorch.org/docs/1.2.0/torchvision/datasets.html?highlight=kinetics#torchvision.datasets.Kinetics400".

**Comment by fuyukaiKUN** [comment]
> hello, i need the kinetics dataset, can you send me that? thank you! [2115996745@qq.com](mailto:2115996745@qq.com)

The original dataset is so large, maybe you can try "https://pytorch.org/docs/1.2.0/torchvision/datasets.html?highlight=kinetics#torchvision.datasets.Kinetics400".

**Comment by cy-kui** [comment]
thanks for your reply! I want to know if I can get the &nbsp;extracted&nbsp; optical flows datasets and frames datasets of kinetics from you or elsewhere .&nbsp; I train or evaluate on a server with 12 GB memory and TITAN Xp GPU, Is it feasible for such a device to conduct pre_training of action recognition on kinetics dataset? If not, is it feasible to train model only on UCF101 and HMDB51 dataset without pre_ training&nbsp; on the kinetics dataset?



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yjxiong/tsn-pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月26日(星期三) 下午2:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yjxiong/tsn-pytorch] Issue about training Kinetics400 (#87)





  
hello, i need the kinetics dataset, can you send me that? thank you! ***@***.***
  
The original dataset is so large, maybe you can try "https://pytorch.org/docs/1.2.0/torchvision/datasets.html?highlight=kinetics#torchvision.datasets.Kinetics400".
